### ISSUE TITLE
Regex expressions and column length not rendered in ClassificationStore editor

### ISSUE THREAD

**Issue by emimontesdeoca** [pimcore/pimcore#8131]
Hello.

You can see on the input field editor that you can change the column length and the regex expression

![image](https://user-images.githubusercontent.com/13499480/107486529-1017a080-6b7d-11eb-9d5b-53eec6aebbe6.png)

But if you go into the classification store and try to change, it's not there.

![image](https://user-images.githubusercontent.com/13499480/107486719-448b5c80-6b7d-11eb-9783-8d0f742e6a89.png)

Is there any reason of why there's no regex expression and column length fields in the classification store editor?

---

I did a small change and it got rendered.

If you change `true` to `false` in `this.editor.setInClassificationStoreEditor();` on the file [keyDefinitionWindow.js ](https://github.com/pimcore/pimcore/blob/5fcf30280c6d95dd8e3a08e9663300b0c5b9b579/bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/keyDefinitionWindow.js#L33), it will display the regex and column length editor in the classification store editor as you can see in the image

![image](https://user-images.githubusercontent.com/13499480/107485994-69cb9b00-6b7c-11eb-98a4-33e23cc1fd9a.png)

This seems to be intended (the default `true`) because if you go into the classes files, for example the input file, you have set to check if it's on the classification store editor as you can see [here](https://github.com/pimcore/pimcore/blob/aafac1f50c1e2f8e0fc39656cd4f12f343d76cc7/bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/input.js#L98) and not render it.

TLDR: Is this intended? Why? Will it break other things?

Thank you.

**Comment by weisswurstkanone** [comment]
@emanmagni , for 
a) yes there is a reason because the way how the data gets persisted is completely different
b) for the regex I do not really see a reason as far as I can remember

Do you  wanna create a PR for b) ?

**Comment by emanmagni** [comment]
@weisswurstkanone Did you mean to tag the OP? @emimontesdeoca

**Comment by weisswurstkanone** [comment]
I meant to bring the regex back to life for classification store.

**Comment by fashxp** [comment]
Dear community, it's been a while since that issue was updated the last time, so we decided to close it for now. If you're still interested in contributing, feel free to add comments here or contact us before creating a pull request to discuss next steps.

Thanks for your understanding.
